src/v/config/configuration.cc

```diff
@@ -3792,6 +3792,23 @@ configuration::configuration()
       {.visibility = visibility::user},
       std::nullopt,
       &validate_non_empty_string_opt)
+  , iceberg_backlog_controller_p_coeff(
+      *this,
+      "iceberg_backlog_controller_p_coeff",
+      "Proportional coefficient for the Iceberg backlog controller. Number of "
+      "shares assigned to the datalake scheduling group will be proportional "
+      "to the backlog size error.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      -0.0003)
+  , iceberg_target_backlog_size(
+      *this,
+      "iceberg_target_backlog_size",
+      "Average size of the datalake translation backlog, per partition, that "
+      "the "
+      "backlog controller will try to maintain.",
```

Comment:
Just want to make sure I understand, does this mean that when we have more than this value not translated, translation is scheduled more aggressively? If so, maybe worth explicitly writing that

Reply:
exactly, added a comment